components/common/circular_chart/circular_chart.tsx

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import './circular_chart.scss';
+
+type CircularChartProps = {
+    value: number;
+    isPercentage: boolean;
+    width: number;
+    height: number;
+    type: 'info' | 'warning' | 'error' | 'success';
```

Comment:
nit: we could probably define a global type for this and use it here and with the chips ... WDYT?

Reply:
I think I rather to keep this isolated this way. Keep the other chip type in the context of the sys console.